src/Passport.js

```diff
@@ -1,20 +1,16 @@
-import React from 'react';
-import Front from './Front';
-import Back from './Back';
-import Middle from './Middle';
+import React from 'react'
+import Front from './Front'
+import Back from './Back'
+import Middle from './Middle'
 
-class Passport extends React.Component {
-    render() {
-        return( 
-            <div>
-                <Front/>
-                <div className='gap'/>
-                <Middle/>
-                <div className='gap'/>
-                <Back/>
-            </div>
-        )
-    }
-}
+const Passport = ({ name, country }) => (
```

Comment:
Since there is no state, you don't need a class, `function` or what they call a `Stateless component` is enough. State is a code smell, i.e. something that should be avoided.

Reply:
Also it is called as `Passport(props)` from `React.createElement`, I destructured `props` so you can see which we're using.